src/qos_core/src/protocol/services/boot.rs

```diff
@@ -295,6 +295,9 @@ impl fmt::Debug for Namespace {
 }
 
 /// The Manifest for the enclave.
+/// NOTE: we currently use JSON format for storing this value.
+/// Since we don't have any `HashMap` inside the `Manifest` it works out of the box.
+/// If we ever do need a map inside, we should use a `BTreeMap` to ensure keys are sorted.
```

Comment:
We should just ban maps all together, even btree since I am not sure how those are serialized